core/frontend/src/main/scala/io/udash/bindings/modifiers/ValueModifier.scala

```diff
@@ -11,18 +11,28 @@ import scalatags.generic._
 
 private[bindings] trait ValueModifier[T] extends Modifier[dom.Element] with Bindings {
   def property: ReadableProperty[T]
-  def builder: (T => Element)
+  def builder: (T => Seq[dom.Element])
   def checkNull: Boolean
   def listen(callback: T => Unit): Registration
 
   override def applyTo(t: dom.Element): Unit = {
-    var element: Element = null
+    var elements: Seq[Element] = null
 
     def rebuild() = {
```

Comment:
This has a lot in common with `ValidationValueModifier.rebuild` - you could probably extract a common trait with this logic.